packages/aor-graphql-client/src/index.js

```diff
@@ -34,16 +34,19 @@ const getOptions = (options, aorFetchType, resource) => {
 
 export default async options => {
     const {
-        client: clientOptions,
+        client: clientObject,
+        clientOptions,
         introspection,
         resolveIntrospection,
         buildQuery: buildQueryFactory,
         override = {},
         ...otherOptions
     } = merge({}, defaultOptions, options);
 
-    const client =
-        clientOptions && clientOptions instanceof ApolloClient ? clientOptions : buildApolloClient(clientOptions);
+    let client = clientObject
+    if(!client){
+      client = clientOptions && clientOptions instanceof ApolloClient ? clientOptions : buildApolloClient(clientOptions);
```

Comment:
Can you remove that and just build the ApolloClient if no client was provided already ?